index.bs

```diff
@@ -90,14 +88,14 @@ The use cases give rise to the following <dfn id="dfn-requirements">requirements
 
 <h2 id="issues">Open issues</h2>
 
-We are <a href="https://github.com/ResponsiveImagesCG/eq-usecases/issues">tracking open issues</a> on Github. Please help us close them!
+We are <a href="https://github.com/WICG/cq-usecases/issues">tracking open issues</a> on Github. Please help us close them!
 
 <h2 id="changes">Change history</h2>
 
-A <a href="https://github.com/ResponsiveImagesCG/eq-usecases/commits/gh-pages">complete history of changes</a> is available on Github.
+A <a href="https://github.com/WICG/cq-usecases/commits/gh-pages">complete history of changes</a> is available on Github.
 
-You can also see all <a href="https://github.com/ResponsiveImagesCG/eq-usecases/issues?q=is%3Aclosed">closed issues</a> relating to this document.
+You can also see all <a href="https://github.com/WICG/cq-usecases/issues?q=is%3Aclosed">closed issues</a> relating to this document.
 
 <h2 id="acks">Acknowledgements</h2>
 
-A <a href="http://www.w3.org/community/respimg/participants">complete list of participants</a> of the Responsive Images Community Group is available at the W3C Community Group Website.
+A <a href="http://www.w3.org/community/respimg/participants">complete list of participants</a> of the Responsive Images Community Group is available at the W3C Community Group Website.
```

Comment:
nit: still talks about RICG here.

Reply:
(also, best to leave the new line at EOF)

Reply:
I've updated this reference, and the mailing list links to point to the WICG now, and added a newline at the end of the file